src/main/java/racingcar/util/CarNames.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.util;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CarNames {
+    private static final String COMMA_DELIMITER = ",";
+    private static final String EXCEPTION_MESSAGE = "유효하지않은 입력입니다.";
+
+    private final String value;
+
+    public CarNames(String carName) {
+        hasNotNUllEmpty(carName);
+        this.value = carName;
+    }
```

Comment:
개인적으로 CarNames 클래스는 util 보다는 domain 성격에 더 가까운 것 같아요! 이 부분에 대해서는 정적 팩토리 메서드를 사용해 보시는 건 어떠신가요!?
```suggestion
    private CarNames(String carName) {
        hasNotNUllEmpty(carName);
        this.value = carName;
    }
    
    public static CarNames create(String carName) {
        return new CarNames(carName);
    }
```

정적 팩토리 메서드는 객체 생성 로직을 클래스 내부에 숨기는 역할 즉, 캡슐화를 통해 코드를 객체 지향적으로 작성하는 데 도움을 주기 때문에 적용해 보는 것도 좋을 것 같아요!

Reply:
주옥같은 피드백 감사합니다 ! 알려주신 내용 공부 해봐야겠네요